corehq/apps/reports/tests/sql_fixture.py

```diff
@@ -27,7 +27,7 @@
 
 
 def load_data():
-    engine = create_engine(make_url(settings.SQL_REPORTING_DATABASE_URL))
+    engine = connection_manager.get_engine('default')
```

Comment:
Is `SQL_REPORTING_DATABASE_URL` always guarenteed to be the same as the default db? If so, can we deprecate that setting?

Reply:
There's django's `default` and then there's the reporting `default`. If `SQL_REPORTING_DATABASE_URL` isn't defined then they are the same. But if it defined then it can be different. 

But here we're getting the reporting `default` which is always the same as `SQL_REPORTING_DATABASE_URL`